app/(main)/chat/[chatId]/components/MessageComposer.tsx

```
@@ -395,8 +409,9 @@ const truncateFilePath = (filePath: string) => {
 const ModelSelection: FC<{
   currentModel: { provider: string; name: string } | null;
   disabled: boolean;
+  currPlan: string | undefined;
   loadCurrentModel: () => {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)